partiql-ast/src/main/resources/partiql_ast.ion

```diff
@@ -563,17 +563,22 @@ select::[
 ]
 
 exclude::{
-  exprs: list::[exclude_expr],
+  exprs: list::[item],
   _: [
-    exclude_expr::{
-      root: '.identifier.symbol',
+    item::{
+      root: '.expr.var',
       steps: list::[step],
     },
     step::[
-      exclude_tuple_attr::{ symbol: '.identifier.symbol' },
-      exclude_collection_index::{ index: int },
-      exclude_tuple_wildcard::{},
-      exclude_collection_wildcard::{},
+      attr::{
+        symbol: '.identifier.symbol'
+      },
+      pos::{
+        index: int
+      },
+      wildcard::{
+        type: [ STRUCT, COLLECTION ]
```

Comment:
Seeing the bigger picture now I'm curious if we want to be consistent on struct verbiage or tuple verbiage. I think you had it great, just drop the extra `exclude_`

```suggestion
      tuple_attr:{ symbol: '.identifier.symbol' },
      tuple_wildcard::{},
      coll_index::{ index: int },
      coll_wildcard::{},
```

But from the customer's perspective a tuple is a struct and this is the public api so "binding tuple" is more a specification term whereas customers interact with "structs"

```
      struct_field:{ symbol: '.identifier.symbol' },
      struct_wildcard::{},
      coll_index::{ index: int },
      coll_wildcard::{},
```

Up to you!

Reply:
discussed offline. will change in `partiql-ast` and `partiql-plan` to use "struct" rather than "tuple" since customers may be more familiar with using "struct" in the public api.

Also discussed keeping thins consistent and using the following for the step variants:
```
struct_field:{ symbol: '.identifier.symbol' },
struct_wildcard::{},
coll_index::{ index: int },
coll_wildcard::{},
```